MAUI/TextInputLayout/States-And-Colors.md

```diff
@@ -0,0 +1,248 @@
+                    </VisualState>
+                    <VisualState Name="Error">
+                        <VisualState.Setters>
+                            <Setter Property="Stroke" Value="#B3261E"/>
+                        </VisualState.Setters>
+                    </VisualState>
+            </VisualStateGroup>
+        </VisualStateGroupList>
+    </VisualStateManager.VisualStateGroups>
+    <Entry />
+</inputLayout:SfTextInputLayout>  
+ 
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "User name";
+inputLayout.ErrorText = "User name available";
+inputLayout.Content = new Entry(); 
+
+VisualStateGroupList visualStateGroupList = new VisualStateGroupList() { };
+
+VisualStateGroup visualStateGroup = new VisualStateGroup();
+
+visualStateGroup.Name = "CommonStates";
+
+VisualState normalState = new VisualState() { Name = "Normal" };
+Setter normalStrokeSetter = new Setter() { Property = StrokeProperty, Value = Color.FromArgb("#79747E") };
+normalState.Setters.Add(normalStrokeSetter);
+
+VisualState foucsedState = new VisualState() { Name = "Focused" };
+Setter focusedStrokeSetter = new Setter() { Property = StrokeProperty, Value = Color.FromArgb("#6750A4") };
+foucsedState.Setters.Add(focusedStrokeSetter);
+
+VisualState errorState = new VisualState() { Name = "Error" };
+Setter errorStrokeSetter = new Setter() { Property = StrokeProperty, Value = Color.FromArgb("#B3261E") };
+errorState.Setters.Add(errorStrokeSetter);
+
+visualStateGroup.States.Add(normalState);
+visualStateGroup.States.Add(foucsedState);
+visualStateGroup.States.Add(errorState);
+
+visualStateGroupList.Add(visualStateGroup);
+
+Setter mainSetter = new Setter() { Property = VisualStateManager.VisualStateGroupsProperty, Value = visualStateGroupList };
+
+var style = new Style(typeof(SfTextInputLayout));
+
+style.Setters.Add(mainSetter);
+
+Resources = new ResourceDictionary() { style };
+
+inputLayout.Style = style;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![States](images/StatesAndColors/States.png)
+
+## Disabled state
+
+The text input layout is disabled by setting the `IsEnabled` property to `false`. The color of the container and other UI elements will also be changed to the disabled state, but its color cannot be customized.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<inputLayout:SfTextInputLayout Hint="Name" 
+                               IsEnabled="false">
+    <Entry />
+</inputLayout:SfTextInputLayout>  
+ 
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Name";
+inputLayout.IsEnabled = false;
+inputLayout.Content = new Entry(); 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Disabled state](images/StatesAndColors/Disabled.jpg)
+
+
+## Container color
+The color of the container can be customized by setting the `ContainerBackgroundColor` property. It is applicable when the `ContainerType` property is set to `Filled` and `Outlined`.
```

Comment:
The color of the container can be customized by setting the `ContainerBackgroundColor` property. It is applicable when the `ContainerType` property is set to `Filled` and `Outlined.`

Reply:
Addressed the changes